sycl/test-e2e/bindless_images/vulkan_interop/sampled_images.cpp

```diff
@@ -169,7 +196,9 @@ bool run_sycl(InteropHandleT inputInteropMemHandle,
     }
   }
   if (validated) {
-    printString("Results are correct!\n");
+    std::cout << "\tTest passed: NDims " << NDims << " NChannels " << NChannels
+              << " image_channel_type "
+              << bindless_helpers::channelTypeToString(CType) << "\n";
```

Comment:
```suggestion
#ifdef VERBOSE_PRINT
    std::cout << "\tTest passed: NDims " << NDims << " NChannels " << NChannels
              << " image_channel_type "
              << bindless_helpers::channelTypeToString(CType) << "\n";
#endif
```
nit: just to keep the output clean and only enable verbose output with the macro `VERBOSE_PRINT`.